src/dlgConnectionProfiles.cpp

```diff
@@ -1670,6 +1682,28 @@ void dlgConnectionProfiles::fillout_form()
         }
     }
 
+
+    mudServer = QStringLiteral("Fierymud");
+    if (!deletedDefaultMuds.contains(mudServer)) {
+        pM = new QListWidgetItem(mudServer);
+        pM->setFont(font);
+        pM->setForeground(QColor(Qt::white));
+        profiles_tree_widget->addItem(pM);
+        if (!hasCustomIcon(mudServer)) {
+            mi = QIcon(
+                    QPixmap(QStringLiteral(":/icons/fiery_mud.png")).scaled(QSize(120, 30), Qt::IgnoreAspectRatio,
```

Comment:
It seems to me that the icon is already of size 120x30 - so you do not need to scale it here, instead you can do the simpler line like is done for Imperian:
````CPP
    mi = QIcon(QStringLiteral(":/icons/fiery_mud.png"));
````